Before: 61a83daecc08f650c09b61445d6bdccae91c65981475713aa4bbbe21c7bd2f32
After: 024467d03e7db9ef40534ffaec1a1fe2233dfb1248e12692445be914b5f514af
Patch: 5bd1b2b84a76442082c98acc281f592811171c6db08276360c75e5a12bf4db48
--- corvid-features/corvid.rb	2012-08-09 12:32:52.820305676 +1000
+++ corvid-features/corvid.rb	2012-08-09 12:32:58.407002725 +1000
@@ -3,11 +3,4 @@
 install {
   copy_file       'Rakefile'
   empty_directory 'lib'
-  #  empty_directory 'wtffff'
-}
-
-update {|ver|
-  if ver == 2
-    empty_directory 'wtffff'
-  end
 }
